packages/appstore/src/components/banners/book-banner/banner-template.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Analytics } from '@deriv-com/analytics';
+import { getAllowedLanguages, Localize } from '@deriv/translations';
+import { LabelPairedCircleChevronDownXlBoldIcon, LabelPairedXmarkLgBoldIcon } from '@deriv/quill-icons';
+import { TEbooksUrl } from 'Components/banners/book-banner/book-banner';
+import { SessionStore } from '@deriv/shared';
+
+type TBookBannerTemplate = {
+    e_books_url: TEbooksUrl;
+    e_book_from_landing: keyof TEbooksUrl;
+    lang: ReturnType<typeof getAllowedLanguages>;
+};
+const onCancel = () => {
+    SessionStore.remove('show_book');
+    document.getElementById('e-book-banner')?.classList.add('hidden');
```

Comment:
Just one nitpick, you can ignore it, but I prefer to use React flow to show/hide components instead of native JS. The rest looks good for me, approved!

Reply:
as you wish, team :)